scylla-server/tests/dataTypes-services.test.ts

```diff
@@ -1,7 +1,13 @@
-import { describe, test, expect } from 'vitest';
-import { getAllDataTypes } from '../src/services/dataTypes.services';
+import { describe, test, expect, beforeEach } from 'vitest';
+import { getAllDataTypes, upsertDataType } from '../src/services/dataTypes.services';
+import prisma from '../src/prisma/prisma-client';
 
 describe('Data Type', () => {
+  beforeEach(async () => {
```

Comment:
we do not want to do this, this will delete all the data types and nodes from the database, instead we want to delete the test nodes and data types we make after each test